plugin/src/main/java/org/opensearch/ml/helper/ModelAccessControlHelper.java

```diff
@@ -143,6 +151,90 @@ public void checkModelGroupPermission(MLModelGroup mlModelGroup, User user, Acti
         }
     }
 
+    public void validateModelGroupAccess(
```

Comment:
I believe I worked on this method before. But somehow I don't see it in the feature branch. Not sure what happened!